_posts/2022-10-09-decentralization-of-zk-rollups.md

```diff
@@ -0,0 +1,192 @@
+
+To note is also the fact that any individual can become a sequencer by posting the L1 transaction. This does have benefits in terms of censorship resistance (if another sequencer refuses to process my transaction, I can just submit my own transaction), but it does come with some drawbacks. One is that due to this uncoordinated nature, it may not be obvious as to which sequencer I should submit my transaction to, in order for it to be confirmed quicker, taking into consideration that larger batches are more likely to be confirmed by aggregators quicker, and that some aggregators may wait to pool more L2 transactions before posting to L1 in order to reduce the per-unit gas cost. Additionally, the action of sequencers posting the L2 transaction data on L1s could have opened up the possibility for an inter-sequencer MEV attack. An observer of the L1 mempool can observe L2 transactions as they come in, and if they identify a transaction that is of potential MEV value they could have submitted another transaction with the same transactions but now with the desired order / inclusion of transactions and a higher gas tip. This means that the attacker's transaction would have been accepted before the honest sequencers’, resulting in the sequencer losing out on the reward of L2 gas, and having to pay for L1 gas without any reimbursement. But, the team has implemented a ChainID function that is specific to each sequencer, and is included in the signed transaction, so that a user only ever authorizes one sequencer (and only them) to execute a transaction, thereby mitigating the chance of a sandwich attack by another sequencer. 
```

Comment:
`But, the team has implemented a ChainID function that is specific to each sequencer, and is included in the signed transaction` - is this copied from some other places? There is no specific project in the context.

Reply:
It is not copy-pasted, a reviewer had mentioned that they had made this change in the implementation to solve this problem. Will make the necessary change.